api/provider/aws/id.go

```diff
@@ -64,3 +65,26 @@ func hashNow() string {
 	salt := time.Now().Format(time.StampNano)
 	return fmt.Sprintf("%x", md5.Sum([]byte(salt)))
 }
+
+func TaskDefinitionARNToECSDeployID(arn string) string {
```

Comment:
the stuff added in this file isn't needed, this is bad code to carryover

Reply:
✅